<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="simulation_output"/>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="run_sim_button">
         <property name="text">
          <string>RUN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_sim_button">
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="sel_def_dir">
         <property name="text">
          <string>Select Definitions Dir</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="sel_psychsim_dir">
         <property name="text">
          <string>Select Psychsim Dir</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="load_sim_button">
         <property name="text">
          <string>Load Sim</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="sim_loaded_state">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="psychsim_dir_path">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="def_dir_path">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="sim_path_label">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="select_sim">
         <property name="text">
          <string>Select Sim</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>simulation</string>
    </property>
    <addaction name="actionSelect_load_config"/>
   </widget>
   <widget class="QMenu" name="menudata">
    <property name="title">
     <string>data</string>
    </property>
    <addaction name="actionview_data"/>
    <addaction name="actionquery_data"/>
    <addaction name="actioncreate_samples"/>
   </widget>
   <widget class="QMenu" name="menuplots">
    <property name="title">
     <string>plots</string>
    </property>
    <addaction name="actioncreate_plot"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menudata"/>
   <addaction name="menuplots"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_load_config">
   <property name="text">
    <string>load config</string>
   </property>
  </action>
  <action name="actionview_data">
   <property name="text">
    <string>view raw data</string>
   </property>
  </action>
  <action name="actionquery_data">
   <property name="text">
    <string>query data</string>
   </property>
  </action>
  <action name="actioncreate_samples">
   <property name="text">
    <string>create samples</string>
   </property>
  </action>
  <action name="actioncreate_plot">
   <property name="text">
    <string>create plot</string>
   </property>
  </action>
  <action name="actionview_samples">
   <property name="text">
    <string>view samples</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
